vcluster/_fragments/sleepmode-istio-example.mdx

```diff
@@ -0,0 +1,118 @@
+import Flow, { Step } from '@site/src/components/Flow';
+import CodeBlock from '@theme/CodeBlock'
+import SleepmodeIstioConfig from '!!raw-loader!@site/vcluster/configure/vcluster-yaml/_code/sleepmode-istio-config.yaml'
+import SleepmodeIstioExample from '!!raw-loader!@site/vcluster/configure/vcluster-yaml/experimental/_code/sleepmode-istio-examples.yaml'
+
+### Configure sleep mode for use with Istio
+
+<Flow id="sleepmode-ingress-example">
+  <Step>
+Create the `kind` cluster.
+
+Here we are pinning the ports 80, 443 and the status port 15021 to specific ports in the Kubernetes node port range.  These
+ are used by Istio Gateway and needed for running this example with `kind`.
+
+```shell title="Create a kind cluster"
+kind create cluster --name istio-demo --config - <<EOF
+kind: Cluster
+apiVersion: kind.x-k8s.io/v1alpha4
+nodes:
+  - role: control-plane
+    extraPortMappings:
+      - containerPort: 30000
+        listenAddress: 127.0.0.1
+        hostPort: 80
+        protocol: TCP
+      - containerPort: 31000
+        listenAddress: 127.0.0.1
+        hostPort: 443
+        protocol: TCP
+      - containerPort: 32000
+        listenAddress: 127.0.0.1
+        hostPort: 15021
+        protocol: TCP
+EOF
+```
+  </Step>
+  <Step>
+    Install istio
+
+    ```shell title="Install istio"
+    istioctl install -y
+    ```
```

Comment:
Should remove since we don't support non-ambient. I've resolved other convo related to this.